modules/test/playwright/tests/frontend-js-aui-web/closeActiveMenu.spec.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {systemSettingsPageTest} from '../../fixtures/systemSettingsPageTest';
+
+export const test = mergeTests(loginTest(), systemSettingsPageTest);
+
+test('@LPD-26435 Close active menu', async ({page}) => {
+	await test.step('Go to wiki', async () => {
+		const openProductMenu = page.getByLabel('Open Product Menu');
+
+		const contentAndData = page.getByRole('menuitem', {
+			name: 'Content & Data',
+		});
+
+		if (await openProductMenu.isVisible()) {
+			openProductMenu.click();
+		}
+
+		await contentAndData.waitFor({state: 'visible'});
+		await contentAndData.click();
+
+		const wiki = page.getByRole('menuitem', {name: 'Wiki'});
+
+		await wiki.waitFor({state: 'visible'});
+		wiki.click();
+	});
+
+	await test.step('Create new wiki', async () => {
+		const newWikiButton = page.getByRole('link', {name: 'Add Wiki'});
```

Comment:
We should avoid adding tests on pages of other product teams. There are two ways around this:
1. We can add this test into the relevant project, not new `frontend-js-aui-web`.
2. We can build a sample module with this case and build a test for it.

I don't think we have a sample module for AUI stuff, and we probably don't want to create one. With that in mind, (1) is probably the right way to go in this case. We should add this to existing `wiki` tests, if they exist, or add them. We should also inform responsible team and ask for a review. @dsanz @herrardo what do you think?

Reply:
I created wiki-web test. If this gets the go ahead I will send to Lima which I believe is responsible for wiki-web.